mozetl/taar/taar_update_whitelist.py

```diff
@@ -0,0 +1,75 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import click
+import json
+import requests
+from taar_utils import store_json_to_s3
+
+
+WHITELIST_FILENAME = "only_guids_top_200"
+ADDON_META_URI = "https://addons.mozilla.org/api/v3/addons/search/?app=firefox&sort=created&type=extension&guid={}"  # noqa
+EDITORIAL_URI = "https://addons.mozilla.org/api/v4/discovery/editorial/"
+
+
+class GUIDError(BaseException):
+    pass
+
+
+def load_raw_json(url):
+    r = requests.get(url)
+    if 200 == r.status_code:
+        # process stuff here
+        json_data = json.loads(r.text)
+        return json_data
+
```

Comment:
This should probably raise an exception, otherwise the script will write out an empty list when the status is not 200.

Reply:
Exceptions and test cases added for this.